mlops/ProjectFile.py

```diff
@@ -3,15 +3,23 @@
 """
 import yaml
 import datetime
+import os
 
 
-class ProjectFile:
+class ProjectFile():
 
-    def __init__(self, config, projectfile_name='MLProject'):
+    def __init__(self, config, path='.', projectfile_name='MLProject', use_localhost=False):
         self.config = config
         self.projectfile_name = projectfile_name
```

Comment:
how is this different from config['project']['NAME']

Reply:
This refers to the project file which has the filename MLProject - this is different to the project name. The MLProject file tells mlflow how to configure the docker environment when running a project